manifests/09_credentials.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
-  name: openshift-manila-csi-driver
+  name: openshift-manila-csi-driver-operator
```

Comment:
Interesting, this was wrong before, right? It should have been `openshift-manila-csi-drivers`